deployment/environment/crib/helpers.go

```diff
@@ -219,3 +219,154 @@ func getTierChainSelectors(allSelectors []uint64, highTierCount int, lowTierCoun
 
 	return orderedSelectors[0:highTierCount], orderedSelectors[highTierCount : highTierCount+lowTierCount]
 }
+
+// FundSolanaAccountsWithLogging requests airdrops for the provided accounts and waits for confirmation.
```

Comment:
nitpick (feel free to ignore)
```suggestion
// fundSolanaAccountsWithLogging requests airdrops for the provided accounts and waits for confirmation.
```